pages/api/upload.js

```diff
@@ -0,0 +1,48 @@
+import formidable from "formidable";
+import cloudinary from "cloudinary";
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+cloudinary.config({
+  cloud_name: process.env.CLOUD_NAME,
+  api_key: process.env.API_KEY,
+  api_secret: process.env.API_SECRET,
+});
+
+export default async function handler(request, response) {
+  switch (request.method) {
+    case "POST":
```

Comment:
here you defined requested method "POST" in uppercase letters but later you use lowercase i dont know if that could lead to conflicts

Reply:
ok i read about it and it is only a convention to be better readable for other developers .it has no effect on the function